src/rpcwallet.cpp

```diff
@@ -310,33 +311,27 @@ Value getaddressesbyaccount(const Array& params, bool fHelp)
 
 void SendMoney(const CTxDestination &address, CAmount nValue, CWalletTx& wtxNew)
 {
+    CAmount curBalance = pwalletMain->GetBalance();
+
     // Check amount
     if (nValue <= 0)
         throw JSONRPCError(RPC_INVALID_PARAMETER, "Invalid amount");
 
-    if (nValue > pwalletMain->GetBalance())
+    if (nValue > curBalance)
         throw JSONRPCError(RPC_WALLET_INSUFFICIENT_FUNDS, "Insufficient funds");
 
-    string strError;
-    if (pwalletMain->IsLocked())
-    {
-        strError = "Error: Wallet locked, unable to create transaction!";
-        LogPrintf("SendMoney() : %s", strError);
-        throw JSONRPCError(RPC_WALLET_ERROR, strError);
-    }
-
```

Comment:
See commit msg ``remove a check for IsLocked() that is done in sendtoaddress and sendfrom RPC calls already``.